source/_integrations/freedompro.markdown

```diff
@@ -3,7 +3,13 @@ title: Freedompro
 description: Instructions for how to integrate Freedompro accessories within Home Assistant.
 ha_category:
   - Light
-ha_release: 2021.7
+  - Switch
+  - Lock
+  - Fan
+  - Binary Sensor
+  - Cover
+  - Climate  
+ha_release: 2021.8
```

Comment:
This seems incorrect...

Reply:
Hi @frenck what's wrong?

Reply:
This integration is added in 2021.7, not 2021.8.